src/Microsoft.AspNetCore.Razor.Evolution/CodeGeneration/DesignTimeCSharpRenderer.cs

```diff
@@ -77,7 +77,13 @@ public override void VisitUsingStatement(UsingStatementIRNode node)
 
         public override void VisitCSharpStatement(CSharpStatementIRNode node)
         {
-            if (node.Source != null)
+            if (node.Source != null && string.IsNullOrWhiteSpace(node.Content))
```

Comment:
True. But if we are not going to have a line mapping, what is the point of writing the whitespace content?